R/kpca_poly.R

```diff
@@ -169,15 +170,15 @@ bake.step_kpca_poly <- function(object, new_data, ...) {
       "predict",
       .ns = "kernlab",
       object = object$res,
-      rlang::expr(as.matrix(new_data[, object$columns]))
+      rlang::expr(as.matrix(new_data[, col_names]))
     )
   comps <- rlang::eval_tidy(cl)
   comps <- comps[, seq_len(object$num_comp), drop = FALSE]
   colnames(comps) <- names0(ncol(comps), object$prefix)
   comps <- as_tibble(comps)
   comps <- check_name(comps, new_data, object)
   new_data <- vec_cbind(new_data, comps)
```

Comment:
```suggestion
  new_data <- vctrs::vec_cbind(new_data, comps)
```

We may be better off doing a global search and replace on this, so I'll stop leaving this comment. :)